clang/include/clang/Driver/Options.td

```diff
@@ -2926,6 +2926,8 @@ defm clangir_direct_lowering : BoolFOption<"clangir-direct-lowering",
 
 def emit_cir : Flag<["-"], "emit-cir">, Visibility<[ClangOption, CC1Option]>,
   Group<Action_Group>, HelpText<"Build ASTs and then lower to ClangIR, emit the .cir file">;
+def emit_flat_cir : Flag<["-"], "emit-flat-cir">, Visibility<[ClangOption, CC1Option]>,
+  Group<Action_Group>, HelpText<"Build ASTs and then lower to ClangIR with no nested regions, then emit the .cir file">;
```

Comment:
I'd change the help text to: `Similar to -emit-cir but also lowers structured CFG into basic blocks.`